bindata/network/frr-k8s/webhook.yaml

```diff
@@ -71,17 +71,13 @@ spec:
         - --webhook-mode=onlywebhook
         - --disable-cert-rotation=true
         - --namespace=$(NAMESPACE)
-        - --metrics-bind-address=:7572
         env:
         - name: NAMESPACE
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
         image: {{.FRRK8sImage}}
-        name: frr-k8s-webhook-server
-        ports:
-        - containerPort: 7572
-          name: monitoring
+        name: frr-k8s-webhook-server        
```

Comment:
I think we'd still need `       - --webhook-port=9443` here as we are hostnetworking the pod and it needs to be a reserved one (for which I still have https://github.com/openshift/enhancements/pull/1815 dangling)